src/excalidraw-embed/webpack.prod.config.js

```diff
@@ -77,9 +77,12 @@ module.exports = {
         test: /\.js($|\?)/i,
       }),
       new webpack.optimize.LimitChunkCountPlugin({
-        maxChunks: 1,
+        maxChunks: 38,
```

Comment:
Where's this magic number coming from?

Reply:
This is the max number of named chunks we can have, increasing more than this leads to `[id].js` chunk which I was trying to avoid. We can further improve the splitting strategy once we refactor out backed code as well.